### PR TITLE
Fixes #1532 : Added a container to couple all the UI elements.

### LINKED ISSUE #1532: Addition of container to couple all the UI elements
**Actual Behaviour**

Presently there is no container used .

**Expected Behaviour**
After addition of container, UI will look clean

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by duttaditya18**
Fixes #1532 

**Changes :**
* Added a container to couple all the UI elements.

Screenshots for the change: 

Screenshot

**Preview Link : **
https://rocky-stream-65090.herokuapp.com/

### CODE REVIEW

**Comment by ParthS007** [comment]
@duttaditya18 looks good, can you change the color of the container ?

File: src/www/css/main.css [hunk 1]
```diff
@@ -46,4 +46,17 @@ pre#buildLog{
...
+	padding: 20px;
+	background: #0000001a;
+	box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.54);
+	margin-top: 40px;
+	margin-bottom: 50px;
+}
+
+div.form-div-board li{
+	color: #101012;
 }
```
Comment (ParthS007) [inline src/www/css/main.css]: add extra line 👍

**Comment by duttaditya18** [comment]
@ParthS007 Sure. Which color do you suggest?

File: src/www/css/main.css [hunk 2]
```diff
@@ -46,4 +46,17 @@ pre#buildLog{
...
+	margin-bottom: 0;
+	padding: 20px;
+	background: #0000001a;
+	box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.54);
+	margin-top: 40px;
+	margin-bottom: 50px;
+}
+
+div.form-div-board li{
+	color: #101012;
```
Comment (geekyd) [inline src/www/css/main.css]: @duttaditya18 Please do not change the color of text.

Comment (duttaditya18) [reply src/www/css/main.css]: Then tell me what should be the color of the background.

File: src/www/css/main.css (hunk 1 above)
Comment (duttaditya18) [reply src/www/css/main.css]: Done.

**Comment by duttaditya18** [comment]
@ParthS007 Done!

**Comment by geekyd** [comment]
@duttaditya18 Can you share deployment link?

**Comment by duttaditya18** [comment]
@geekyd So I would need to host it on Heroku? That'd take a lot of time. This is a simple change. You can also download it from my Repo.

**Comment by ParthS007** [comment]
@duttaditya18 It hardly takes 2-3 minutes, @geekyd  which color is right for box container, should we go with white or any other?

**Comment by duttaditya18** [comment]
@ParthS007 One click deployment doesn't work. I think the current color look good. Rest is up to you!

**Comment by geekyd** [comment]
@duttaditya18 For heroku deployment go to your forked repo & click on Deploy to heroku link here https://github.com/duttaditya18/open-event-webapp#how-do-i-deploy-web-app-generator-with-heroku

@ParthS007 It should be black, in recent changes it was changed to white.

**Comment by duttaditya18** [comment]
@geekyd Here is the Heroku Log. It doesn't work.

```
-----> Node.js app detected

-----> Creating runtime environment

       

       NPM_CONFIG_LOGLEVEL=error

       NPM_CONFIG_PRODUCTION=true

       NODE_VERBOSE=false

       NODE_ENV=production

       NODE_MODULES_CACHE=true

-----> Installing binaries

       engines.node (package.json):  unspecified

       engines.npm (package.json):   unspecified (use default)

       

       Resolving node version 8.x...

       Downloading and installing node 8.9.4...

       Using default npm version: 5.6.0

-----> Restoring cache

       Skipping cache restore (not-found)

-----> Building dependencies

       Installing node modules (package.json + shrinkwrap)

       npm ERR! code E404

       npm ERR! 404 Not Found: timed-out@https://registry.npmjs.org/timed-out/-/timed-out-4.0.1.tgz

       

       npm ERR! A complete log of this run can be found in:

       npm ERR!     /app/.npm/_logs/2018-01-06T18_39_17_010Z-debug.log

-----> Build failed

       

       We're sorry this build is failing! You can troubleshoot common issues here:

       https://devcenter.heroku.com/articles/troubleshooting-node-deploys

       

       Some possible problems:

       

       - Node version not specified in package.json

       https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version

       

       Love,

       Heroku

       

 !     Push rejected, failed to compile Node.js app.

 !     Push failed
```

**Comment by ParthS007** [comment]
please run `npm install` again in your repo directory

**Comment by duttaditya18** [comment]
@ParthS007 All checks have passed. It is up to you now.

**Comment by ParthS007** [comment]
@duttaditya18 You can go through this https://github.com/duttaditya18/open-event-webapp/blob/development/docs/INSTALLATION_HEROKU.md and follow manual deployment

**Comment by ParthS007** [comment]
@duttaditya18 I appreciate your work but for approval, preview link is required.

**Comment by duttaditya18** [comment]
@ParthS007 I did all that. I know how to set up a Heroku app. This doesn't work. Or at least I cannot get it to work.

You can download this from  my repo and go to `src\www\index.html` to preview the changes.

The PR Template did not mention any Preview Link. If you suggest any changes I will do so.

**Comment by duttaditya18** [comment]
@ParthS007 @geekyd Here is the preview link. Check it out : https://rocky-stream-65090.herokuapp.com/

**Comment by duttaditya18** [comment]
@fossasia/open-event-web-app Travis CI failed because of an error in `npm install`. It is not related to my PR. Can anyone check?

**Comment by duttaditya18** [comment]
So I updated my branch. It works now!

**Comment by duttaditya18** [comment]
@ParthS007 Thanks! Getting Heroku set up is indeed not that hard. Sorry for being impatient the other day. 😄

**Comment by duttaditya18** [comment]
@geekyhd Thanks for helping me with this!

**Comment by geekyd** [comment]
@duttaditya18 Great job! :)

**Pull request merged**